src/objects/core/zcl_abapgit_objects_files.clas.abap

```diff
@@ -88,6 +80,9 @@ CLASS zcl_abapgit_objects_files DEFINITION
     METHODS get_file_pattern
       RETURNING
         VALUE(rv_pattern) TYPE string .
+    METHODS is_aff
```

Comment:
rename to `is_json` ?

Reply:
is_aff seems to be more precise as looking just at the method name is_json might also include my_file.w3mi.data.json

Reply:
the method doesn't look into the content of the `json` to determine if it's indeed `aff`. how about `is_json_metadata`?

Reply:
yea

Reply:
done